oneview/resource_server_profile.go

```diff
@@ -1317,6 +1317,22 @@ func resourceServerProfile() *schema.Resource {
 													Type:     schema.TypeBool,
 													Optional: true,
 												},
+												"login_priv": {
+													Type:     schema.TypeBool,
+													Optional: true,
```

Comment:
I think in documentation we need to make it clear that these properties only for Gen10, Gen10+ servers. Otherwise these could lead to idempotency issues, we should also clear about our examples

Reply:
Done